pages/index.tsx

```diff
@@ -1,12 +1,36 @@
 import type { NextPage } from "next";
 import Link from "next/link";
-import { Container } from "react-bootstrap";
+import { Button, Container, Image } from "react-bootstrap";
+import React from "react";
 
 const Home: NextPage = () => {
   return (
-    <Container>
-      <h1>Hello, world!</h1>
-      <Link href="/about">About</Link>
+    <Container className="text-center">
+      <Image
+        src="/blueno.png"
+        alt="Blueno logo"
+        width="300"
+        height="300"
+        className="d-inline-block align-top"
+      />
+      <h1>Wordify</h1>
+      <Link href="/game">
+        <Button variant="light" size="lg">
+          Play As Guest
+        </Button>
+      </Link>
+      <br />
+      <Link href="/#">
+        <Button variant="light" size="lg">
+          Log In
+        </Button>
+      </Link>
+      <br />
+      <Link href="/profile">
```

Comment:
Not sure if there should be a profile button on the home page. When I envisioned the app, it made sense to just have the "play as guest" and "log in" buttons on the home screen. If we were to include a profile screen, what would it include? How can we track user statistics and profile picture when they have not signed in yet?